packages/flutter_goldens_client/test/skia_client_test.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I do not recall why, but all the tests for the client are with the rest of the flutter_goldens tests. There is already a FakeProcessManager etc. there if you want to move them there to share code. 

Then again, it does make more sense to test the client.. in the client package. 🙃

Reply:
Weird. Why are they two packages then? Should we merge them? There's no README for either so I'm not sure I really understand the intended use of these packages...

Reply:
I think I'm going to land this as-is for now and we can talk about what we want to do with these packages separately. Either way we're going to have to follow-up here (either this helps the flakiness or not, and if it does, we'll want to more aggressively do this approach in other places in this file).